app/views/layouts/_policies_acknowledgement.html.erb

```diff
@@ -0,0 +1,19 @@
+<% if current_user.present? && !current_user.policies_acknowledged? %>
```

Comment:
IMHO this guard should be moved into layout and simplified by moving the logic into helper (`show_policies_acknowledge_banner?`). Same for hammy one.